src/main/java/service/url_redirect/UrlRedirectPresenter.java

```diff
@@ -0,0 +1,14 @@
+package service.url_redirect;
+
+/**
+ * Gives information from the interactor to the User.
+ */
+public class UrlRedirectPresenter implements UrlRedirectOutputBoundary {
+
+    /**
+     * Shows error messages if opening the URL fails.
+     * @param message message to be displayed
+     */
+    public void displayError(String message) {
```

Comment:
If you are planning to implement this in the future, you should add a TODO here so it can be found easily.